librina/src/netlink-parsers.cc

```diff
@@ -6387,6 +6394,12 @@ parseIpcmRegisterApplicationRequestMessage(nlmsghdr *hdr) {
 	                      nla_get_u16(attrs[IRAR_ATTR_REG_IPC_ID]));
 	}
 
+	if (attrs[IRAR_ATTR_BLOCKING]) {
+		result->blocking = true;
+	} else {
+		result->blocking = false;
+	}
+
```

Comment:
I think attrs[IRAR_ATTR_BLOCKING] is a pointer. It may be
result->blocking = attrs[IRAR_ATTR_BLOCKING] != NULL;